app/api/domains/cho.py

```diff
@@ -1214,9 +1221,16 @@ async def handle(self, player: Player) -> None:
                             # use player mode if not specified
                             mode_vn = player.status.mode.as_vanilla
 
+                        # parse the mods from regex
+                        if r_match["mods"] is not None:
```

Comment:
i generally don't like usage of if/else blocks like this

nesting is ugly and i try to keep it to a minimum, so i would do
```py
mods = None
if condition:
    mods = val
```

Reply:
well I did it the same way it's on other parts in cho.py, just copied existing if statements and put them in place